server/constants/prisonerCategories.ts

```diff
@@ -0,0 +1,22 @@
+import { PrisonerCategories } from '../data/visitSchedulerApiTypes'
+
+const prisonerCategories: Readonly<Record<PrisonerCategories, string>> = {
+  A_EXCEPTIONAL: 'A EXCEPTIONAL',
+  A_HIGH: 'A HIGH',
+  A_PROVISIONAL: 'A PROVISIONAL',
+  A_STANDARD: 'A STANDARD',
+  B: 'B',
```

Comment:
Might be worth checking (with Simon?) if these are what the 'human readable' category labels should be. E.g. in the staff UI we render `C` as `Cat C`

Reply:
Agreed. Will speak to Simon.

Reply:
👍 fine to leave as it is for now and we can tweak in separate PR if he wants them changing